pkg/domain/infra/abi/containers_runlabel.go

```diff
@@ -180,13 +180,28 @@ func generateRunlabelCommand(runlabel string, img *libimage.Image, inputName str
 }
 
 func replaceName(arg, name string) string {
+	if arg == "NAME" {
+		return name
+	}
+
 	newarg := strings.ReplaceAll(arg, "$NAME", name)
-	return strings.ReplaceAll(newarg, "${NAME}", name)
+	newarg = strings.ReplaceAll(newarg, "${NAME}", name)
+	if strings.HasSuffix(newarg, "=NAME") {
```

Comment:
Is it possible to have a trailing space after NAME here?  If so, we probably want to trim it.

Reply:
No these are arguments, only if the user wrapped something with "foo=NAME " would that happen, and in that case I think we should ignore.